kidsability/src/containers/Login.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import "./Login.css";
+
+export default function Login() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  function validateForm() {
+    return email.length > 0 && password.length > 0;
```

Comment:
can use this library to do the validation https://www.npmjs.com/package/react-email-validator

Reply:
validateForm checks to make sure both aren't null; i added the email-validate to the logic. Lmk what you think